help/destinations/catalog/crm/salesforce.md

```diff
@@ -10,7 +10,7 @@ exl-id: bd9cb656-d742-4a18-97a2-546d4056d093
 
 [!DNL Salesforce CRM](https://www.salesforce.com/) is a popular Customer Relationship Management (CRM) platform.
```

Comment:
Couple of comments: 

1. Incorrect bracketing. When using DNL within a link, we should double-bracket, like below: 
`[[!DNL Salesforce CRM]](https://www.salesforce.com/)`

One of the brackets is for the DNL, the other is for hyperlinking

2. Is there a more precise link that we can use instead of the generic link to Salesforce.com? Not sure if linking to the Salesforce homepage provides much value.